app/views/api_keys/index.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="page-header">
-<%= form_for(@api_key) do |f| %>
+<%= form_for(:api_key) do |f| %>
```

Comment:
`@api_key` is only used to create a button that creates a new API key when pressed.  The `@api_key` created in the controller always gets garbage collected.  I found this when I got `spec/views/api_keys/index.html.erb_spec.rb` working, before I decided to throw out the boilerplate.  I figure it's better this way anyway because we're not creating an object that we don't need.